.github/workflows/ci-pull.yml

```diff
@@ -1,10 +1,8 @@
 name: ci
 
 on:
+  workflow_dispatch:
```

Comment:
Do we still need this change? Things _seem_ to work without it...

Reply:
This allows you to run a workflow manually on any branch. IMO it's useful and doesn't break anything.

Reply:
Allows us to kick off builds without actually running a change, manually. Can be really useful!

Reply:
OK, thanks for the explanation.

Then yeah, that should be done as a separate commit (it's not related to the other changes).

Reply:
The file renaming part of this commit should be dropped.